src/main/java/com/br/luminous/entity/Device.java

```diff
@@ -36,4 +41,6 @@ public void setPower(float power) {
     public void setUsageTime(LocalTime usageTime) {
         this.usageTime = usageTime;
     }
+
+    public Address getAdress(){return adress;}
```

Comment:
you don't need to create getters and setters because you're using @Data anotation